scripts/climateMetrics/climateRarity.yml

```diff
@@ -42,50 +46,19 @@ inputs:
     type: float
     example: 0.25
   time_span:
-    label: time span
-    description: period climate projection
-    type: text
+    label: Future time span
+    description: period for climate projection, start year to end year
+    type: options
+    options:
+      - 2011-2040 
+      - 2041-2070
+      - 2071-2100
     example: 2041-2070
-  use_obs:
-    label: use obs
-    description: use observations
-    type: boolean
-    example: TRUE 
-  species:
-    label: species names
-    description: scientific name of the species
-    type: text
-    example: "Glyptemys insculpta"
-  species_year_start:
-    label: year start
-    description: starting year to load observations
-    type: int
-    example: 1980
-  species_year_end:
-    label: year end
-    description: ending year to load observations
-    type: int
-    example: 2020
-  species_limit:
-    label: limit
-    description: maximum number of observations to load
-    type: int
-    example: 2000
   bbox:
-    label: bbox coordinates
-    description: coordinates of the bbox, in the order xmin, xmax, ymax, ymin.
+    label: Bounding box
+    description: Vector of float, bbox coordinates of the bbox in the order xmin, ymin, xmax, ymax
     type: float[]
     example: [-2316297,-1971146,1015207,1511916]
-  buffer_box:
-    label: buffer box
-    description: buffer surronding box
-    type: int
-    example: 0
-  shapefile_path:
-    label: shapefile path
-    description: Path to a .shp file representing the extent.
-    type: application/dbf
-    example: "/output/studyExtent/studyExtent_R/a3cb46ded0b02d361f7450f439dace86/study_extent.shp"
 outputs:
   output_tif:
     label: output metric
```

Comment:
What is the unit of this output? Maybe we should put that in the description... Percentage? Degrees? I have no clue! But with a range from -1 to 91, I hope it's not degrees ;)

Reply:
updated